src/string_bytes.cc

```diff
@@ -54,7 +54,7 @@ class ExternString: public ResourceType {
       return scope.Escape(String::Empty(isolate));
 
     TypeName* new_data =
-        static_cast<TypeName*>(malloc(length * sizeof(*new_data)));
+        static_cast<TypeName*>(node::Malloc(length * sizeof(*new_data)));
```

Comment:
A future improvement is to replace Malloc calls like this one with a Calloc-like version that does overflow checks (but doesn't zero the memory.)